activities/communication/index.md

```diff
@@ -29,15 +30,15 @@ Phone numbers and lines are managed in our VOIP supplier Voys' [online managemen
 
 In order to clearly convey who is communicating, this template for email signature could be used. It has no styling in order to make it more probable that email clients and mailing lists will handle the signature gracefully.
 
-```
+``` example
```

Comment:
What is the purpose of these? Should they maybe be headers instead of inside the code block?

Reply:
The delinter wanted the code in the codeblock to be identified.

Reply:
There are already headers that mark these out as examples.

Reply:
That should not be a thing, also this is not the correct syntax so should not solve it :)

```suggestion
```
```